projects/f3d.bundle.common.cmake

```diff
@@ -14,7 +14,7 @@ set(CPACK_PACKAGE_VENDOR "${f3d_name}-app")
 set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "${f3d_description}")
 set(CPACK_PACKAGE_HOMEPAGE_URL ${f3d_url})
 set(CPACK_PACKAGE_CHECKSUM MD5)
-set(CPACK_PACKAGE_EXECUTABLES f3d f3d)
+set(CPACK_PACKAGE_EXECUTABLES f3d)
```

Comment:
this is breaking the Windows CI

Reply:
` CPack Error: CPACK_PACKAGE_EXECUTABLES should contain pairs of <executable> and <icon name>.`

Reply:
indeed